table/get_context.h

```diff
@@ -53,6 +63,19 @@ class GetContext {
   };
   GetContextStats get_context_stats_;
 
+  // Constructor
```

Comment:
Nit: maybe these comments can be moved next to member variable declaration? since all the constructor does is simple assignment (except seq_)?

Reply:
I'm not sure. The member variables are private and not meant to be looked at. I think these can be here so anyone instantiating the object can understand what the arguments are meant for.